cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -56,12 +57,12 @@ import qualified Hedgehog.Extras.Test.Base as H
 import qualified Hedgehog.Extras.Test.File as H
 
 -- | Returns JSON encoded hashes of the era, as well as the hard fork configuration toggle.
-createConfigYaml :: ()
+createConfigJSON :: ()
```

Comment:
Small nitpick, [acronyms should be camelcased this way](https://github.com/input-output-hk/cardano-node-wiki/wiki/Style-guide#naming-convention):
```suggestion
createConfigJson :: ()
```

Reply:
Thanks for the pointer @carbolymer :+1: Changing.